app/assets/stylesheets/components/_accordion.scss

```diff
@@ -0,0 +1,109 @@
+.app-c-accordion {
+  &.js-accordion-with-descriptions {
+    margin-bottom: $gutter;
+    border-bottom: 1px solid $border-colour;
+  }
+}
+
+.app-c-accordion__header {
+  position: relative;
+  padding-top: 14px;
+  padding-bottom: 12px;
+
+  .js-accordion-with-descriptions & {
+    border-top: 1px solid $border-colour;
+    cursor: pointer;
+
+    &:hover {
+      background: $grey-3;
+    }
+  }
+}
+
+.app-c-accordion__icon {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  height: 16px;
+  width: 16px;
+  margin-top: -8px;
+  background-image: image-url('icons-plus-minus.png'); // PNG fallback for SVG
+  background: image-url("icons-plus-minus.svg"), linear-gradient(transparent, transparent); // SVG sprite
+  background-repeat: no-repeat;
+  background-position: 0 -16px;
+
+  @include media (tablet) {
+    right: 15px;
+  }
+
+  .section-is-open & {
+    background-position: 0 0;
+  }
+}
+
+.app-c-accordion__title {
+  @include core-24;
+
+  font-weight: bold;
+}
+
+// removes extra dotted outline from buttons in Firefox
+// on focus (standard yellow outline unaffected)
+@mixin firefox-remove-second-button-outline {
+  &::-moz-focus-inner {
+    border: 0;
+  }
+}
+
+.app-c-accordion__title-button {
```

Comment:
You could de-duplicate some of the styles between `title-button` and `controls-button`

These are all shared:
```
@include firefox-remove-second-button-outline;
color: $link-colour;
cursor: pointer;
background: none;
border: 0;
```